packages/cli-platform-ios/src/commands/runIOS/index.ts

```diff
@@ -49,7 +49,19 @@ async function runIOS(_: Array<string>, ctx: Config, args: FlagsT) {
   let {packager, port} = args;
 
   // check if pods need to be installed
-  await resolvePods(ctx.root, ctx.dependencies, {forceInstall: args.forcePods});
+  if (ctx.project.ios?.automaticPodsInstallation) {
+    await resolvePods(ctx.root, ctx.dependencies, {
+      forceInstall: args.forcePods,
+    });
+  } else if (args.forcePods && !ctx.project.ios?.automaticPodsInstallation) {
+    logger.warn(
+      `${chalk.bold(
+        '--force-pods',
+      )} has no effect because automatic CocoaPods installation is disabled. In order to use this flag, set ${chalk.bold(
+        'project.ios.automaticPodsInstallation',
+      )} to true in ${chalk.bold('react-native.config.js')}.`,
```

Comment:
💡 We can link docs here, so user will be able to read more about this option in config.